toc.yml

```diff
@@ -196,6 +196,9 @@
       - name: Network Programming in the .NET Framework
         tocHref: /dotnet/framework/network-programming/
         topicHref: /dotnet/framework/network-programming/index
+      - name: .NET Framework Deployment
```

Comment:
we also need to add the entries that are under the Development Guide to a node with that name. 

line 190 should also be changed from "Migration Guide to the .NET Framework 4.6 and 4.5" to just "Migration Guide"